components/elm/cime_config/config_component.xml

```diff
@@ -83,6 +83,12 @@
       <value compset="SSP585_CAM5%CMIP6_ELM*_BGC%B">2015-2100_SSP585_CMIP6bgc_transient</value> 
       <value compset="2010_CAM5%CMIP6-LR_ELM">2010_CMIP6LR_control</value>
       <value compset="2010_CAM5%CMIP6-HR_ELM">2010_CMIP6HR_control</value>
+      <value compset="2010_CAM5.*SCREAM-LR" >2010_CMIP6_control</value>
+      <value compset="2010_CAM5.*SCREAM-HR" >2010_CMIP6_control</value>
+      <value compset="2010_CAM5.*SCREAM-LR-DYAMOND2" >2010_dy2scream_control</value>
```

Comment:
This looks like it's using a new elm use_case file, but I don't see such a use case file added in this PR. Is this a mistake, or maybe the use case files didn't get committed?

Reply:
Thanks for catching this. It did get left off the commit.